src/components/AnnotationCreator.jsx

```diff
@@ -229,16 +301,49 @@ export default function AnnotationCreator({
               ),
             ];
             const theta = get(rect, 'theta', 0);
+            const newEncounterGuids = pending
+              ? []
+              : await createEncounter(
```

Comment:
so, instead of doing createEncounter and then postAnnotation, it's very important that we only continue moving through this process if the first request worked correctly. otherwise we could easily wind up with random new annotations that shouldn't be there.

plus, if you wait for success before moving on to the next step, you should never have multiple errors, only one error at a time. so it should simplify your life to not have to deal with an array of errors

Reply:
Because pending sightings (i.e., AGSs) ought to be able to create orphan annotations, we need to allow orphan annotation creation to be possible.

Your point about only one error at a time is well-taken.